apps/staking/app/mystakes/page.tsx

```diff
@@ -21,11 +18,10 @@ export async function generateMetadata() {
 }
 
 export default function Page() {
-  const chain = useChain();
   return (
     <ModuleGrid size="lg" className="h-full px-4 md:px-10 xl:auto-rows-auto">
       <div className="md:max-h-screen-without-header col-span-1 flex h-full min-h-max flex-col gap-4 pb-8 md:overflow-y-auto md:overflow-x-hidden">
-        <ModuleGrid>
+        <ModuleGrid className="mr-1">
```

Comment:
Is this always on the right or do you want to use me-1?

Reply:
always right